pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -657,7 +659,7 @@ spec:
     action: list
     options: |-
       labelSelector: openebs.io/replica=jiva-replica,openebs.io/persistent-volume={{ .Volume.owner }}
-    retry: "12,10s"
+    retry: "24,5s"
```

Comment:
we should be checking for consistent number of replica pods to be deployed at least to move forward

Reply:
Done